app/src/main/java/woowacourse/paint/Line.kt

```diff
@@ -0,0 +1,33 @@
+package woowacourse.paint
+
+import android.graphics.Paint
+import android.graphics.Path
+import android.graphics.PathMeasure
+
+data class Line(
+    val path: Path,
+    val brush: Paint
+) {
+
+    val length: Float
+        get() = PathMeasure(path, false).length
```

Comment:
```suggestion
val length: Float = PathMeasure(path, false).length
```
위와 같이 수정해도 똑같이 동작이 되던데, get을 쓰신 이유가 무엇인가요? 정말 순수 100% 궁금해서 물어봅니다!😄

Reply:
뽀또가 제시해주신 방법으로 생성하는게 적합합니다!! 
생성된 Line 객체의 length가 가변적이라고 생각했는데 온전히 제 실수입니다ㅎㅎ